dotnet/docfx.json

```diff
@@ -51,7 +51,10 @@
       "apiPlatform": "dotnet",
       "author": "dotnet-bot",
       "breadcrumb_path": "~/breadcrumb/toc.yml",
-      "extendBreadcrumb": true
+      "extendBreadcrumb": true,
+      "feedback_system": "GitHub",
+      "feedback_github_repo": "aspnet/EntityFramework.ApiDocs",
```

Comment:
I am having second thoughts on this one. Assuming it can work, there is an argument for pointing this to EntityFramework.Docs: we (the EF product team) don’t ever triage issues here at EntityFramework.ApiDocs. I expect issues here to be mostly about problems with the way the API docs get generated and not the contents (that is, the kind of issues @dend would probably look into). Thoughts?

Reply:
It’s up to you. For .NET, we send all issues to the same repo. A lot of the feedback is re. APIs (missing info, missing samples, bugs and also issues with the way the API docs get generated). We don’t even have to enable for API if you think it’s not worth it. We’re still experimenting in our side.

Reply:
Ok, let’s point to EntityFramework.Docs (analog to what the .NET team does). We can always redirect issues here when needed. Thanks.